From: Tommy Webb <tommy@calyxinstitute.org>
Date: Tue, 5 Mar 2024 21:19:56 +0000
Subject: Virtual keyboard resizes layout by default

Needed with Viewport Size Protection, or the keyboard will cover up
text boxes while typing.

Copied from uazo/cromite Move-navigation-bar-to-bottom.patch.

Issue: calyxos#2135
Original License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html

Change-Id: I56c19972ac6a96ba4bd695e61f3443e496fcdf81
---
 chrome/browser/prefs/browser_prefs.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -2174,7 +2174,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry,
 
 #if BUILDFLAG(IS_ANDROID)
   registry->RegisterBooleanPref(prefs::kVirtualKeyboardResizesLayoutByDefault,
-                                false);
+                                true);
 #endif
 
 #if BUILDFLAG(ENABLE_SCREEN_AI_SERVICE)
-- 

